openshift/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM centos:centos7
```

Comment:
How about using `jboss/base-jdk:8` ? or even a CE image if that's internal?

Reply:
I think using the base-jdk image makes sense, and I'd prefer not to use an internal CE image

Reply:
Yeah. I will change the base image.

Reply:
Actually. RadarGun is not a JBoss project per se. It's an independent project so I'm not sure basing it on JBoss is a good step. I'd rather not.

Reply:
agree with @mgencur, as RG is cited as "standard testing tool", we shouldn't tie it to any specific project unless necessary.

Reply:
For me, I'm not too worried about using a JBoss image. I'm more concerned if the JBoss image includes the items that are being installed in this Dockerfile. If the JBoss image doesn't include anything extra except for Java, then I agree it's not worth changing.